### DISCUSSION TITLE
Image ghosting

### DISCUSSION THREAD

**Discussion by rickylars** [volotat/SD-CN-Animation#77] [Q&A] [answered]
Hi everyone,

I'm currently trying to use the vid2vid to recreate a video based on the original video I have. However, I'm running into an issue where on the first few frames of the output video, both the image created from my prompt and the original video are superimposed on top of each other. Specifically, on the first few frames, the image from the prompt is stronger while the video is like a ghost over it, and then it is reversed.

This happens with ControlNet turned on and off. 

Any help or advice would be greatly appreciated! Thank you in advance.

**Comment by volotat** [comment, marked as answer]
Check out methods described here: https://github.com/volotat/SD-CN-Animation/discussions/80#discussioncomment-5879239 I hope it will work better.

**Comment by rickylars** [comment]
That's much better now, thank you. I also updated to the latest version (0.9).
